go/ql/lib/semmle/go/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -241,9 +241,15 @@ module Public {
     }
 
     /**
-     * Holds if the summary is auto generated.
+     * Holds if the summary is auto generated and not manually generated.
      */
     predicate isAutoGenerated() { none() }
+
+    /**
+     * Holds if the summary has the given provenance where `true` is
+     * `generated` and `false` is `manual`.
+     */
+    predicate hasProvenance(boolean generated) { summaryElement(this, _, _, _, generated) }
```

Comment:
This implementation of the member predicate ought to be on `SummarizedCallableExternal`.